packages/react/README.md

```diff
@@ -1,25 +1,26 @@
 # Set up the Faro React distribution of the Faro Web SDK
 
-Faro-React is a distribution of the Faro Web SDK for project using React, which offers easier integrations and the following features:
+The Faro Web-SDK is a highly configurable open source real user monitoring (RUM) library built on OpenTelemetry and integrating seamlessly with Grafana Cloud and Grafana Frontend Observability. Faro-React is a distribution of the Faro Web SDK for project using React, which offers easier integrations and the following features:
```

Comment:
@grafsean 
Faro is not built on OpenTelemetry. 
It uses it's own protocol and has nothing in common with  openteletry.

Only exception is the `web-tracing` package which is a wrapper around otel which makes it easier to setup otel.js and to use it with Faro.

Reply:
Is the use of OTel through web-tracting automatic and not optional?

Reply:
It's automatic.
The web-tracing package is a wrapper around otel.js, so opentelemetry is not optional for that specific package.
It's meant to make including otel easier.